ts/src/renderer/phaser/classes/PhaserAnimatedEntity.ts

```diff
@@ -11,18 +11,48 @@ class PhaserAnimatedEntity extends PhaserEntity {
 
 		const bounds = entity._bounds2d;
 		const sprite = this.sprite = scene.add.sprite(0, 0, key);
+
 		sprite.setDisplaySize(bounds.x, bounds.y);
 		sprite.rotation = entity._rotate.z;
 
 		Object.assign(this.evtListeners, {
-			'play-animation': entity.on('play-animation', this.playAnimation, this)
+			'play-animation': entity.on('play-animation', this.playAnimation, this),
+			size: entity.on('size', this.size, this),
 		});
 	}
 
 	protected playAnimation (animationId: string): void {
 		this.sprite.play(`${this.key}/${animationId}`);
 	}
 
+	protected transform (data: {
+		x: number;
+		y: number;
+		rotation: number
+	}): void {
+		this.sprite.setPosition(data.x, data.y);
+		this.sprite.rotation = data.rotation;
```

Comment:
sprite rotation and sprite setDisplaySize should stay the same, yeah? @pavle-goloskokovic

Reply:
position should be applied to `gameObject` container, rotation and scale to animated texture, or rather `sprite`